external/js/cothority/lib/misc/misc.js

```diff
@@ -36,6 +36,45 @@ function hexToUint8Array(hex) {
     });
 }
 
+/**
```

Comment:
I really really really hate having a lib/misc module here. misc and/or util packages are dumping grounds for all the things we haven't thought carefully about.

I'm asking myself two things: 1. can this compare code be in the test directory? 2. surely there is a well-known module in the NPM universe for uint8ArrayToHex and hexToUint8Array (for god's sake, they even had a module called "left_pad"!)

I know I'm being a Go prima-donna here. If you and @gnarula tell me "this is just how it in JS" I'll believe you and shut up.

Reply:
> I really really really hate having a lib/misc module here. misc and/or util packages are dumping grounds for all the things we haven't thought carefully about.

While it may not be the best thing to do, it is still tested and kept under control. If you want I can rename the package "bitutils" to make it clearer ? I'm not sure how it's done in JS but relying on an external library for something as trivial as that does not seem like better to me. The less dependency the better.